docs/source/conf.py

```diff
@@ -33,6 +33,7 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'tools.sphinxext.c7n_schema',
```

Comment:
tools isn't a package itself, its a directory which contains independent subtools that maybe packages (pretty much all the subdirs prefixed c7n_ are packages).